swap_meet/vendor.py

```diff
@@ -1,2 +1,125 @@
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory=None):
+        if not inventory:
+            inventory = []
+        self.inventory = inventory
```

Comment:
Nice handling of the `None` default value! Another way we could write this is with a ternary operator:
```suggestion
        self.inventory = inventory if inventory else []
```
This can be read as "If the parameter inventory is truthy, assign self.inventory to inventory, else set self.inventory to an empty list". The general blueprint for a ternary operator looks like: `variable_name = <value_if_condition_true> if <condition_statement> else <value_if_condition_false>`.
More info: https://book.pythontips.com/en/latest/ternary_operators.html